Source/ACE.Server/WorldObjects/Player_Combat.cs

```diff
@@ -209,9 +215,6 @@ public DamageEvent DamageTarget(Creature target, WorldObject damageSource)
         /// </summary>
         public override void OnDamageTarget(WorldObject target, CombatType attackType, bool critical)
         {
-            if (critical)
```

Comment:
with this move, critical emote will no longer happen when a CombatPet or other damage source hits a creature, so that is something to keep in mind